### PR TITLE
Critical fixes and (slight) polish

### LINKED ISSUE #80: File Availability
The user should be notified or there should be some indicator when an uploaded file is ready to be downloaded.

### PR DESCRIPTION
**Pull request by seveibar**
fixes #80 #81 #64 #70

### CODE REVIEW

**Comment by seveibar** [comment]
Autoupdating isn't included in this version

**Comment by seveibar** [comment]
Autoupdating message now included, essentially a very bad version of #38

**Comment by seveibar** [comment]
Please review this pull request retroactively

File: site/scripts/ui/views/files.js [hunk 1]
```diff
@@ -16,30 +18,98 @@ ui._files = (function(){
             ui._uploadFile.setPrivacy("public");
             ui.switchView("upload-file");
         });
+        eSearch.keydown(function(e){
+            if (e.keyCode == 13){
```
Comment (lukechampine) [inline site/scripts/ui/views/files.js]: just wondering, are there constants defined anywhere that map keys to their keycode?

File: site/scripts/ui/views/files.js [hunk 2]
```diff
@@ -16,30 +18,98 @@ ui._files = (function(){
...
+        files.forEach(function(fileObject){
+            var fileNickname = fileObject.Nickname;
+            if (!searchValue || fileNickname.indexOf(searchValue)!=-1){
+                var blocksRemaining = fileObject.TimeRemaining;
                 var eFile = eFileBlueprint.clone().removeClass("blueprint");
+                var available = fileObject.Available;
                 eFile.find(".name").text(fileNickname);
-                eFile.find(".size").text("? MB");
+                eFile.find(".size").text(" "); //TODO we can't get size ATM
+                eFile.find(".time").text(blocksRemaining + "t"); //TODO this unit sucks
```
Comment (lukechampine) [inline site/scripts/ui/views/files.js]: you can give a rough estimate by multiplying blocksRemaining * 10 minutes

File: site/scripts/controller/controller.js [hunk 3]
```diff
@@ -2,9 +2,13 @@ var controller = (function() {
 
     var data = {};
     var createdAddressList = [];
+    var dataListeners = {};
     var uiConfig;
 
     function init() {
+        $.get("http://www.siacoin.com/betamessage.txt", function(content){
```
Comment (lukechampine) [inline site/scripts/controller/controller.js]: I assume this file is going to be added to gh-pages at some point?

**Pull request merged**